pkg/types/types.go

```diff
@@ -72,6 +73,14 @@ type Address []byte
 // upkeep at a block number
 type UpkeepKey []byte
 
+func (u UpkeepKey) BlockKeyAndUpkeepID() (blockKey string, upkeepID string) {
```

Comment:
I was thinking of calling this function something like `Explode` or `Components` or `IDs`, but I thought the longer, more descriptive name made it easier for the client functions to use